manual/source/classes/contributed_definitions/ellipsometry-structure.rst

```diff
@@ -0,0 +1,179 @@
+.. _Ellipsometry-Structure:
+
+========================
+Optical Spectroscopy
+========================
+
+.. index::
+   Ellipsometry
+   DispersiveMaterial
+
+
+.. _Ellipsometry:
+
+Ellipsometry
+##############
+
+Ellipsometry is an optical characterization method to describe optical properties of interfaces and thickness of films.
+The measurements are based on determining how the polarization state of light changes upon transmission and reflection.
+Interpretation is based on Fresnel equations and numerical models of the optical properties of the materials.
+
+In the application definition we provide a minimum set of description elements allowing for a reproducible recording of ellipsometry measurements. 
+
+
+Application Definitions
+-----------------------
+
+    :ref:`NXopt`:
```

Comment:
Actually, Optical Spectroscopy experiments, but like in case of XPS, ARPES, the community use here an acronym, OPT. On the other hand, in text we use the long version for clarity.